packages/reports/src/sass/app/modules/smart-chart.scss

```diff
@@ -70,6 +70,40 @@
 
 /** @define chart-spot-label */
 .chart-spot-label {
+    &-profit {
+        position: relative;
+        top: -2.8rem;
+        left: 2.5rem;
+
+        .chart-spot-label__value-container {
+            padding: 0.8rem;
+            line-height: normal;
+            border-radius: 0.6rem;
+        }
+
+        &--lost {
+            &:before {
+                border-bottom: 0.5rem solid var(--text-loss-danger);
+            }
+        }
+
+        &--won {
+            &:before {
+                border-bottom: 0.5rem solid var(--text-profit-success);
+            }
+        }
+
+        &:before {
+            content: '';
+            position: absolute;
+            top: 50%;
+            left: -1rem;
```

Comment:
ditto

Reply:
This is same as I mentioned above: `left: -1rem` is not equal to `right: 1 rem`. We should also do positioning based on left and top borders since the element always starts rendering from there, while right and bottom borders will be changing its location based on content width and height